subprojects/internal-testing/src/main/groovy/org/gradle/util/TestPrecondition.groovy

```diff
@@ -38,14 +38,8 @@ enum TestPrecondition implements org.gradle.internal.Factory<Boolean> {
     NO_FILE_PERMISSIONS({
         !FILE_PERMISSIONS.fulfilled
     }),
-    SET_ENV_VARIABLE({
-        !UNKNOWN_OS.fulfilled
-    }),
     WORKING_DIR({
-        !UNKNOWN_OS.fulfilled && JavaVersion.current() < JavaVersion.VERSION_11
-    }),
-    PROCESS_ID({
-        !UNKNOWN_OS.fulfilled
+        JavaVersion.current() < JavaVersion.VERSION_11
```

Comment:
It's a bit weird. Wasn't `System.getProperty("user.dir")` available before? Or do we use this precondition for something else?

Reply:
I did not dig deep, only checked that a test using this precondition fails without it.